packages/core/src/node/core-process-manager.spec.ts

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { Container } from 'inversify';
+import { CoreProcessManager } from './core-process-manager';
+import { expect } from 'chai';
+
+/** PPID, PID */
+const mockPsOutput = `\
```

Comment:
I've added a comment below, but I believe that children have a PID lower than their parent's PID, and then the function might fail, so it would be good to have a testcase that reflects that.

Reply:
Good catch, I updated the test.